tests/cases/unittests/session.ts

```diff
@@ -0,0 +1,459 @@
+/// <reference path="..\..\..\src\harness\harness.ts" />
+
+module ts.server {
+	let lastWrittenToHost: string,
+	mockHost: ServerHost = {
+		args: [],
+		newLine: '\n',

```

Comment:
Again, will do - but tslint didn't capture this style guideline when I ran it over the file. It'd be nice to see it added for the future.